### PR TITLE
Split hand pin change for splinky v3

### PR DESCRIPTION
**Pull request by AlexBaldwin42**
Moved the MASTER_RIGHT and the commented-out code for the SPLIT_HAND_PIN and SPLIT_HAND_PIN_LOW_IS_LEFT to the respective v2 v3 config.h files as the pin was changed from v2 to v3.

### CODE REVIEW

File: keyboards/bastardkb/charybdis/3x5/v2/splinky/v2/config.h [hunk 1]
```diff
@@ -25,3 +25,11 @@
 #define SPI_SCK_PIN GP18
 #define SPI_MOSI_PIN GP19
 #define POINTING_DEVICE_CS_PIN GP14
+
+/* Handedness. */
+#define MASTER_RIGHT
+
+// To use the handedness pin, resistors need to be installed on the adapter PCB.
+// If so, uncomment the following code, and undefine MASTER_RIGHT above.
+// #define SPLIT_HAND_PIN GP15
```
Comment (0xcharly) [inline keyboards/bastardkb/charybdis/3x5/v2/splinky/v2/config.h:34]:
```suggestion
// #define SPLIT_HAND_PIN GP13
```

Comment (0xcharly) [inline keyboards/bastardkb/charybdis/3x5/v2/splinky/v2/config.h:34]:
```suggestion
// #define SPLIT_HAND_PIN GP13
```

File: keyboards/bastardkb/charybdis/3x5/v2/splinky/v3/config.h [hunk 2]
```diff
@@ -25,3 +25,11 @@
 #define SPI_SCK_PIN GP22
 #define SPI_MOSI_PIN GP23
 #define POINTING_DEVICE_CS_PIN GP16
+
+/* Handedness. */
+#define MASTER_RIGHT
+
+// To use the handedness pin, resistors need to be installed on the adapter PCB.
+// If so, uncomment the following code, and undefine MASTER_RIGHT above.
+// #define SPLIT_HAND_PIN GP13
```
Comment (0xcharly) [inline keyboards/bastardkb/charybdis/3x5/v2/splinky/v3/config.h:34]:
```suggestion
// #define SPLIT_HAND_PIN GP15
```

File: keyboards/bastardkb/charybdis/3x6/v2/splinky/v2/config.h [hunk 3]
```diff
@@ -25,3 +25,12 @@
...
 #define POINTING_DEVICE_CS_PIN GP14
+
+/* Handedness. */
+#define MASTER_RIGHT
+
+// To use the handedness pin, resistors need to be installed on the adapter PCB.
+// If so, uncomment the following code, and undefine MASTER_RIGHT above.
+// #define SPLIT_HAND_PIN GP13
+// #define SPLIT_HAND_PIN_LOW_IS_LEFT  // High -> right, Low -> left.
+
```
Comment (0xcharly) [inline keyboards/bastardkb/charybdis/3x6/v2/splinky/v2/config.h:36]:
```suggestion
```

File: keyboards/bastardkb/charybdis/3x6/v2/splinky/v3/config.h [hunk 4]
```diff
@@ -25,3 +25,12 @@
...
 #define POINTING_DEVICE_CS_PIN GP16
+
+/* Handedness. */
+#define MASTER_RIGHT
+
+// To use the handedness pin, resistors need to be installed on the adapter PCB.
+// If so, uncomment the following code, and undefine MASTER_RIGHT above.
+// #define SPLIT_HAND_PIN GP15
+// #define SPLIT_HAND_PIN_LOW_IS_LEFT  // High -> right, Low -> left.
+
```
Comment (0xcharly) [inline keyboards/bastardkb/charybdis/3x6/v2/splinky/v3/config.h:36]:
```suggestion
```

File: keyboards/bastardkb/charybdis/4x6/v2/splinky/v2/config.h [hunk 5]
```diff
@@ -25,3 +25,12 @@
...
 #define POINTING_DEVICE_CS_PIN GP14
+
+/* Handedness. */
+#define MASTER_RIGHT
+
+// To use the handedness pin, resistors need to be installed on the adapter PCB.
+// If so, uncomment the following code, and undefine MASTER_RIGHT above.
+// #define SPLIT_HAND_PIN GP13
+// #define SPLIT_HAND_PIN_LOW_IS_LEFT  // High -> right, Low -> left.
+
```
Comment (0xcharly) [inline keyboards/bastardkb/charybdis/4x6/v2/splinky/v2/config.h:36]:
```suggestion
```

File: keyboards/bastardkb/charybdis/4x6/v2/splinky/v3/config.h [hunk 6]
```diff
@@ -25,3 +25,12 @@
...
 #define POINTING_DEVICE_CS_PIN GP16
+
+/* Handedness. */
+#define MASTER_RIGHT
+
+// To use the handedness pin, resistors need to be installed on the adapter PCB.
+// If so, uncomment the following code, and undefine MASTER_RIGHT above.
+// #define SPLIT_HAND_PIN GP15
+// #define SPLIT_HAND_PIN_LOW_IS_LEFT  // High -> right, Low -> left.
+
```
Comment (0xcharly) [inline keyboards/bastardkb/charybdis/4x6/v2/splinky/v3/config.h:36]:
```suggestion
```

**Review by 0xcharly — CHANGES_REQUESTED** [review]
Thanks for the PR!

Would you mind also doing this for the Scylla, TBK mini and Skeletyl? It would be great to have all these in a single PR.

**Comment by AlexBaldwin42** [comment]
No problem! I will do those as well.

**Pull request merged**